Tests/Marketplace/marketplace_services.py

```diff
@@ -2673,7 +2666,7 @@ def set_pack_dependencies(self, packs_dependencies_mapping, packs_dict, marketpl
 
         self._user_metadata[Metadata.DEPENDENCIES] = first_level_dependencies
         self._first_level_dependencies = first_level_dependencies
-        self._all_levels_dependencies = all_levels_dependencies
+        self._all_levels_dependencies = all_levels_dependencies_dict
```

Comment:
`self._all_levels_dependencies`should get a list, not a dict

Reply:
It doesn't matter, we take only the keys in the usage

Reply:
Ok, so please fix the default value in the pack __init__ as well